apps/esbuild/shell/main.ts

```diff
@@ -1,5 +1,7 @@
 //@ts-nocheck
 
 (async () => {
-  await import('./bootstrap');
+  setTimeout(() => {
```

Comment:
still need to figure this out, init of my lifecycle doesnt happen sync and init itself is not awaitable right now.